tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -192,4 +196,232 @@ public function testProManagerCannotPutBranchInReview(): void
             $this->assertSame($e->getMessage(), 'You don\'t have access to the resource.');
         }
     }
+
+    public function testMergeWorkflow(): void
+    {
+        $oldBranches = $this->branches->listBranches();
+        $this->assertCount(1, $oldBranches);
+
+        $componentId = 'wr-db';
+        $configurationId = 'main-1';
+        $components = new \Keboola\StorageApi\Components($this->getDefaultBranchStorageApiClient());
+
+        $configuration = (new \Keboola\StorageApi\Options\Components\Configuration())
+            ->setComponentId($componentId)
+            ->setConfigurationId($configurationId)
+            ->setName('Main')
+            ->setDescription('some desc');
+        $components->addConfiguration($configuration);
+
+        $newBranch = $this->branches->createBranch('aaaa');
+
+        $devBranchComponents = new \Keboola\StorageApi\Components($this->getBranchAwareClient($newBranch['id'], [
```

Comment:
proc to delas takto? kvuli branchAwareClientovi?

dal bych tam 
```
new Components($this->getDeveloperStorageApiClient());
```

Reply:
Jo potrebujem updatnut config v branchi, mozno by to malo ist uz do `setUp` ale zatial som to nechal len tu.